mscore/editelement.cpp

```diff
@@ -60,8 +60,8 @@ void ScoreView::updateGrips()
 
             // updateGrips returns grips in page coordinates,
             // transform to view coordinates:
-
-            const QPointF pageOffset(editData.element->findAncestor(ElementType::PAGE)->pos());
+            Element* el = editData.element->findAncestor(ElementType::PAGE);
+            const QPointF pageOffset(el ? el->pos() : QPointF());
```

Comment:
Change to:

`const QPointF pageOffset((page ? page : editData.element)->pos());`

Reply:
same here

Reply:
(same here)